packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -91,13 +91,15 @@ describe('Slider', () => {
       expect(onChange).toBeCalledTimes(1);
       expect(onChange.mock.calls[0][1]).toEqual({ value: 50 });
       expect(inputRef.current?.value).toEqual('50');
-      expect(wrapper.find('.thumb-wrapper').props().style?.transform).toEqual('translateX(45%)');
+      // TODO: Find a way to get thumbWrapper styles to update with simplified prop merging change.
+      // expect(wrapper.find('.thumb-wrapper').props().style?.transform).toEqual('translateX(45%)');
```

Comment:
I guess that we can use [`getComputedStyle()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle):

```tsx
const { getByTestId } = render(<Slider />);

const element = getByTestId();
const styles = getComputeStyle(element);

expect(styles.transform).toEqual();
```

Reply:
I tried adding `getComputedStyles` but it still didn’t seem to work with finding the incoming css transform value. However, the track width is still updating in the tests so I swapped to checking the track position instead of the thumb. The test will still work the same as before now.